src/io/flutter/run/SdkFields.java

```diff
@@ -77,7 +88,8 @@ VirtualFile chooseWorkDir(VirtualFile launchFile, Project project) throws Runtim
     // Work directory is optional; if not set we must infer it.
     if (!StringUtil.isEmptyOrSpaces(workDir)) {
       return checkWorkDir(workDir);
-    } else {
+    }
+    else {
```

Comment:
No objection to updating the formatter.

> This is Google style so I keep forgetting to do it the Jetbrains way.

Yeah and you shouldn't have to.  You should always just format and move on.